app/src/main/java/de/tum/in/tumcampusapp/activities/PersonsSearchActivity.java

```diff
@@ -130,8 +129,21 @@ public void onStartSearch(String query) {
      */
     @Override
     public void onLoadFinished(PersonList response) {
-        if (response.getPersons() == null) {
+        //String "no search result" not shown
+        //change response.getPersons() == null to response.getPersons().isEmpty() or response.getPersons().size() == 0
+        if (response.getPersons().isEmpty()) {
             lvPersons.setAdapter(new NoResultsAdapter(this));
+        } else if (response.getPersons().size() == 1){
+            ListAdapter adapter = new ArrayAdapter<>(this, android.R.layout.simple_list_item_1, android.R.id.text1, response.getPersons());
+            lvPersons.setAdapter(adapter);
+            Person person = (Person) lvPersons.getItemAtPosition(0);
+            // store selected person ID in bundle to get in in StaffDetails
+            Bundle bundle = new Bundle();
+            bundle.putSerializable("personObject", person);
+            // show detailed information in new activity
+            Intent intent = new Intent(this, PersonsDetailsActivity.class);
+            intent.putExtras(bundle);
+            startActivity(intent);
```

Comment:
This looks like copypasta'd 🍝  code to me.

I'd just reuse the onItemClick method here, but moving the duplicated code to a separate method would probably be better.